core/src/com/fallenflame/game/FlareModel.java

```diff
@@ -46,6 +47,11 @@ public float getLightRadius() {
         return lightRadius;
     }
 
+    /**
+     * @return the color of the Flare's tint
+     */
+    public Color getLightColor() {return new Color(255, 255, 255, 0.0f);}
```

Comment:
Can you add this color as a parameter in the global json? That way we have all the parameters consolidated for easy adjustment.